libraries/chain/controller.cpp

```diff
@@ -1627,7 +1629,7 @@ struct controller_impl {
                        emit(self.accepted_transaction, trx);
                    }
 
-                   dmlog_applied_transaction(trace);
+                   dmlog_applied_transaction(trace, &trx->packed_trx()->get_signed_transaction());
```

Comment:
`trn`

Reply:
I don't understand what you mean?

Reply:
There is already the `trn` variable which is the `signed_transaction` in scope.

Reply:
done, thank you Kevin!